tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "compilerOptions": {
     "target": "es2017",
-    "module": "es6",
-    "moduleResolution": "Node",
+    "module": "commonjs",
```

Comment:
This is the big cross-cutting change here. The shared typescript compile will now emit commonjs instead of esm.